components/bl-video-recorder-component/README.md

```diff
@@ -0,0 +1,41 @@
+# Video Recorder
+
+Video Recorder is a component of Backendless UI-Builder designer. This allows to record screen and use the recorded file.
+
+
+<p align="center">
+  <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
+</p>
+
+## Properties
+
+| Property    | Type       | Default Value    | Logic           | Data Binding | UI Setting | Description                                                  |
+|-------------|------------|------------------|-----------------|--------------|------------|--------------------------------------------------------------|
+| Width       | *Number*   | 720              |                 | NO           | YES        | Controls width of component, in 'px'.                        |
+| Height      | *Number*   | 480              |                 | NO           | YES        | Controls height of component, in 'px'.                       |
```

Comment:
I assume we should make it bindable for a case if the app could be opened in mobile and desktop